wsl-setup

```diff
@@ -87,8 +87,10 @@ if [ -n "${local_app_data}" ]; then
   cp "/usr/share/fonts/truetype/ubuntu/UbuntuMono[wght].ttf" "${fonts_dir}"
 fi
 
-# Wait for cloud-init to finish.
-cloud-init status --wait > /dev/null 2>&1 || true
+# Wait for cloud-init to finish if systemd and its service is enabled.
+if `systemctl is-enabled --quiet cloud-init.service 2>/dev/null`; then
```

Comment:
In general, I don’t like to use backtyicks when we havemultiple commands running simultaneously, wdyt about this rather, as we are already relying on some bashism:

```suggestion
if [ systemctl is-system-running --quiet && systemctl is-enabled --quiet cloud-init.service 2>/dev/null ]; then
```
(need to be tested)

Reply:
actually, this won’t work, because "degraded" state will return 1. What do you get when it’s not running at all? (to prevent lookup at the output?

I did try:
`if systemctl is-system-running --quiet && systemctl is-enabled --quiet cloud-init.service 2>/dev/null; then`